Backend/customer/register

nakagame opened the pull request:
[Backend/customer/register] - Created Customer Register
- Make Customer model.
- Modified Login blade

nakagame committed:
Merge branch 'Table/Create/240323Part2' of github.com:kredo-school/18tha-gj_mall into Table/Create/240325

nakagame committed:
[Backend/Customer/Register] - Created Customer Register Function
- Created a new model - Customer
- Modified Auth/RegisterController
- Modified views/resource/register.blade.php

nakagame committed:
Merge branch 'main' into Backend/Customer/Register

kredo-school commented on app/Http/Controllers/Auth/RegisterController.php:
```diff
@@ -49,24 +49,39 @@ public function __construct()
+            'last_name'  => $data['last_name'],
+            'email'      => $data['email'],
+            'password'   => Hash::make($data['password']),
+            // Have to remove payment_id
```
Do you still need to remove the payment_id?

Ninit29 commented on app/Http/Controllers/Auth/RegisterController.php:
```diff
@@ -49,24 +49,39 @@ public function __construct()
+            'email'      => $data['email'],
+            'password'   => Hash::make($data['password']),
+            // Have to remove payment_id
+            'payment_id' => 1,
```
Do you still need the `payment_id` ? If not please remove

Ninit29 commented on app/Models/Users/Customer.php:
```diff
@@ -0,0 +1,21 @@
+        'last_name',
+        'email',
+        'password',
+        'payment_id',
```
Do you still need the `payment_id` ? If not please remove

nakagame committed:
[BackEnd/Customer/Register] - Delete unnecessary code.
- Remove payment_id from Customer model
- Remove payment_id from Register Controller
